components/prism-elixir.js

```diff
@@ -36,6 +40,10 @@ Prism.languages.elixir = {
 		lookbehind: true,
 		alias: 'symbol'
 	},
+  'atom-module': {
+    pattern: /\b[A-Z][A-Z_a-z0-9]*\b/,
```

Comment:
This is intentional as it is meant to highlight modules, e.g.: `Elixir.Namespace` in `Elixir.Namespace.function()`
I tried calling it `module` which is what makes sense, but if I do, it does not tokenize properly, any suggestions here?

Reply:
We do have the `namespace` class of modules and package names.

> I tried calling it `module` which is what makes sense, but if I do, it does not tokenize properly

That's because [another `module` rule](https://github.com/PrismJS/prism/blob/4d7d9b3879e2106e8c9facf04e3ebd73a66f0a1d/components/prism-elixir.js#L89) is inserted overwriting yours. Since your new pattern is essentially a superset of the old one, you could probably remove the old one.

On that note: Why does the old `module` pattern use `[A-Z][\w.\\]+`? I understand the `.` but I don't get the `\\`. Is that even correct?

Reply:
The only thing that comes to mind is elixir's default parameters, but it tokenizes correctly with or without `\\`.

```elixir
def get_time(timestamp \\ 0) do
  # ...
end
```

Reply:
In that case, `\` was probably incorrect.